utilities/openemr-cmd/openemr-cmd-h

```diff
@@ -69,5 +76,9 @@ case "$FIRST_ARG" in
 		openemr-cmd -h
 		;;
 		*)
-		openemr-cmd -h|grep -i $FIRST_ARG
+		# Quoted variable
+		openemr-cmd -h | grep -i "${FIRST_ARG}"
+		;;
 esac
+
+exit 0 # Explicit success exit at end of script
```

Comment:
Was this a shellcheck suggestion? It seems unnecessary, as the script will always exit early if there's an error (due to `set -e` at the top).

Reply:
I just added that to be clear what exist status was used since there are a bunch of custom statuses set in various places in the scripts. It's not necessary.